tests/unittest/unittest.bash

```diff
@@ -0,0 +1,856 @@
+#!/usr/bin/env bash
```

Comment:
Not for this PR. However, we should create a Bazel module that exposes this. We could host the repo under `bazel-contrib`.